Classes/Integration/MultipleItemsProcFunc.php

```diff
@@ -32,7 +33,7 @@ public static function register(string $table, string $field, ?string $additiona
         $GLOBALS['TCA'][$table]['columns'][$field]['config']['itemsProcFunc'] = $newFunction;
     }
 
-    public function execute(array &$parameters, FormDataProviderInterface $formDataProvider): void
+    public function execute(array &$parameters, ItemProcessingService | FormDataProviderInterface $formDataProvider): void
```

Comment:
Unfortunately I can't accept code with union types - Flux still has to support TYPO3v10 which means supporting PHP 7.4 which sadly doesn't support union types. If this method has to support two different classes as second argument, we have to remove the type and add a PHPDOC instead (which only describes this single argument and notes the two possible classes as type; we don't PHPDOC things that are implied by strict types).